build.gradle

```diff
@@ -18,6 +18,18 @@ allprojects {
   }
 }
 
+ext {
```

Comment:
This's cool. However travis CI got failed due to your changes. Could you please update the .travis.yml to fix it before I merge? Just updating build tool version or something.
https://travis-ci.org/nntuyen/mkloader